### PR TITLE
Feature/full screen map

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
This looks very different from when we worked on it this morning. Let me know if you have any additional questions. We can chat on Monday!

File: app/scripts/views/home/home-controller.js [hunk 1]
```diff
@@ -38,6 +39,7 @@
             ctl.search = search;
             $scope.$on('imls:vis:ready', function (e, vis, newMap) {
                 map = newMap;
+                ctl.map = map;
```
Comment (moradology) [inline app/scripts/views/home/home-controller.js]: I'm not sure the local variable is doing anything here, consider removing.

**Comment by moradology** [comment]
+1 on travis' passing

File: app/scripts/views/home/home-controller.js (hunk 1 above)
Comment (CloudNiner) [inline app/scripts/views/home/home-controller.js]: Good catch, there was this and some other dead code, just removed.

**Pull request merged**